cmd/legacy/run.go

```diff
@@ -26,18 +26,21 @@ func Run(v *viper.Viper) {
 		log.Fatalf("failed to load log params: %s", err)
 	}
 
+	var logFile *os.File
+
 	if !logfileParams.ToStdout {
 		log.Debugf("log to file %s", logfileParams.File)
 
-		f, err := os.OpenFile(logfileParams.File, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0666)
+		logFile, err := os.OpenFile(logfileParams.File, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0666)
 		if err != nil {
 			log.Fatalf("error opening log file: %s", err)
 		}
 
-		log.SetOutput(f)
+		log.SetOutput(logFile)
 	}
 
 	log.SetVerbose(logfileParams.Verbose)
+	log.SetJww(logfileParams.Verbose, logFile)
```

Comment:
`logFile` will be nil, if `logfileParams.ToStdout` equals `true`. Maybe passing `logfileParams.ToStdout` into `log.SetJww()` and handle this inside? There are different possibilities..

Reply:
```Go
func SetJww(verbose, w io.Writer) {
	if verbose {
		jww.SetLogThreshold(jww.LevelDebug)
		if w != nil {
			jww.SetLogOutput(w)
		}
	}
}
```